src/renderer/App.tsx

```diff
@@ -60,18 +81,37 @@ export default function App() {
           element: <Show />,
         },
         {
-          path: RouteDefinitions.GameHost,
+          path: RouteDefinitions.GameHost.route,
           element: <GameHost />,
         },
         {
           path: RouteDefinitions.GameTeam,
           element: <GameTeam />,
         },
+        {
+          path: RouteDefinitions.GameSettings,
+          element: <GameSettings />,
+        },
       ],
     },
   ]);
 
   setNavigate(router.navigate);
 
+  // This callback is used to track previous location route of host app flow. When user is on game-host pages, and decides to go outside of
+  // game (back button or home button), team users should be navigated to their home screen too. The 'quitGame' message is send to them for
+  // this purpose.
+  router.subscribe((x) => {
```

Comment:
hm, hard to read. Is that needed to remember previous path, if host clicked "Back" button ? Do we need back button in game at all ? Me personally would leave just one button in game header screen -> finish game. 
But since we already have, let's keep it. It is nice to have feature, which can be useful in some cases.